### PR TITLE
Fix / Settings nav notifications

### PR DESCRIPTION
**Pull request by ronronscelestes**
## What

Fixed settings nav notifications UI + added test snapshots

#12729

### CODE REVIEW

File: packages/core/admin/admin/src/hooks/useSettingsMenu/init.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const init = (initialState, { settings, shouldUpdateStrapi }) => {
   // Sort the links by name
   const sortedGlobalLinks = sortLinks([...pluginsGlobalLinks, ...globalLinks]).map(link => ({
     ...link,
-    hasNotification: link.id === 'application-infos' && shouldUpdateStrapi,
+    hasNotification: link.id === '000-application-infos' && shouldUpdateStrapi,
```
Comment (HichamELBSI) [inline packages/core/admin/admin/src/hooks/useSettingsMenu/init.js:13]: Why `000-`?

Comment (ronronscelestes) [reply packages/core/admin/admin/src/hooks/useSettingsMenu/init.js:13]: changed made by @soupette to make sure this link always show first if I'm not mistaken!

Comment (ronronscelestes) [reply packages/core/admin/admin/src/hooks/useSettingsMenu/init.js:13]: I would let @soupette give his opinion on it!
From what I understand we want the global links to be sorted alphabetically except for the Overview/Application page which should always be first. Currently we are sorting pluginsGlobalLinks + globalLinks with lodash's sortBy on link.id. Those 2 objects are not built in the same place (pluginsGlobalLinks is generated inside StrapiApp) 🤔

Comment (HichamELBSI) [reply packages/core/admin/admin/src/hooks/useSettingsMenu/init.js:13]: Oh ok I understand. It's ok for me!

**Review by HichamELBSI — APPROVED** [review]
LGTM

**Review by MarionLemaire — APPROVED** [review]
Tested on Chrome using Getstarted and admin running on Ubuntu : notification is shown about upgrading Strapi version in Settings section (dot icon after Overview).

**Pull request merged**